__tests__/app.test.js

```diff
@@ -41,4 +41,63 @@ describe("GET /api/topics", () => {
     .get("/api/topis")
     .expect(404)
   })
+})
+
+describe("GET /api/articles/article_id", () => {
+  test('200: returns an array of first article', () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({body}) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(1);
+        articles.forEach((obj) => {
```

Comment:
I wouldn't expect a _single_ resource to be sent back in an array. Rather a key of `article` with a value of the single article